Tracker/Statistics/StatisticsViewController.swift

```diff
@@ -0,0 +1,140 @@
+//
+//  StatisticsViewController.swift
+//  Tracker
+//
+//  Created by macOS on 28.05.2023.
+//
+
+import UIKit
+
+final class StatisticsViewController: UIViewController {
+    let titleLabel = UILabel()
+    let emptyStatisticsView = UIStackView()
+    let emptyStatisticsImage = UIImageView()
+    let emptyStatisticsLabel = UILabel()
+    let stackView = UIStackView()
+    let trackersCompleteBackgroundView = UIView()
+    let trackersCompleteView = UIView()
+    let trackersCompleteCount = UILabel()
+    let trackersCompleteLabel = UILabel()
+    
+    let trackerStore = TrackerStore()
+    let trackerRecordStore = TrackerRecordStore()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        setUI()
+    }
+    
+    override func viewDidAppear(_ animated: Bool) {
+        try? trackersCompleteCount.text = String(trackerRecordStore.getCompletedTrackersCount())
+        
+        setGradientOnView(trackersCompleteBackgroundView)
+    }
+    
+    private func setGradientOnView(_ view: UIView) {
+        let gradient = CAGradientLayer()
+        gradient.frame = CGRect(origin: .zero, size: CGSize(width: view.frame.size.width, height: view.frame.size.height))
```

Comment:
Привет! Строка становится длинновата. Строгого ограничения на количество символов - нет, но коллегам, с  небольшими экранами, это может затруднить чтение кода. 
Я пользуюсь SwiftLint, который, по умолчанию, рекомендует длину строки не более 120 символов.
Предлагаю разбить строку на несколько, руководствуясь CodeStyle'ом:
https://google.github.io/swift/#line-wrapping

Например, вот так:
```suggestion
        gradient.frame = CGRect(origin: .zero,
                                                    size: CGSize(width: view.frame.size.width, height: view.frame.size.height))
```